Missing data version file

After downloading the dataset, I'll run the following: 
```
> python3 -m minerl.viewer MineRLTreechop-v0 v1_absolute_grape_changeling-7_14600-16079

Traceback (most recent call last):
  File "/Users/albertge/anaconda3/lib/python3.7/site-packages/minerl/data/version.py", line 12, in assert_version
    assert os.path.exists(version_file), "more"
AssertionError: more
```

The source tells me that it's looking for a file called 'data/VERSION' (`data` here is the placeholder for the absolute directory). 

`$MINERL_DATA_ROOT` has been set to the correct data directory. And because the dataset folders have been changed, i.e. from `r2g1...` to `v1_...` I'm sure this is an issue, even after downloading the new dataset. 

FWIW, commenting out the line seems to work as advertised. I'm able to control the viewer without issue so far.

--- a/minerl/data/version.py
+++ b/minerl/data/version.py
@@ -23,9 +23,12 @@ def assert_version(data_directory):
             assert DATA_VERSION <= txt, "more"
             assert DATA_VERSION >= txt, "less"
         else:
-            fs = (glob.glob(os.path.join(data_directory, '*-*/*')))
-            for f in fs:
-                assert_prefix(f)
+            for exp in os.listdir(data_directory):
+                if VERSION_FILE_NAME not in exp:
+                    exp_dir =  os.path.join(data_directory, exp)
+                    for f in os.listdir(exp_dir):
+                        assert_prefix(os.path.join(exp_dir, f))
+                
     except AssertionError as e:
         _raise_error(e, data_directory)
 

--- a/minerl/data/version.py
+++ b/minerl/data/version.py
@@ -24,7 +24,7 @@ def assert_version(data_directory):
             assert DATA_VERSION >= txt, "less"
         else:
             for exp in os.listdir(data_directory):
-                if VERSION_FILE_NAME not in exp:
+                if 'MineRL' in exp:
                     exp_dir =  os.path.join(data_directory, exp)
                     for f in os.listdir(exp_dir):
                         assert_prefix(os.path.join(exp_dir, f))

--- a/minerl/data/version.py
+++ b/minerl/data/version.py
@@ -10,10 +10,6 @@ def assert_version(data_directory):
 
     try:
         if os.path.exists(version_file):
-            1/0
-            
-
-            assert os.path.exists(version_file), "more"
             with open(version_file, 'r') as f:
                 try:
                     txt = int(f.read())

--- a/minerl/data/version.py
+++ b/minerl/data/version.py
@@ -1,6 +1,6 @@
 import os
 import re
-
+import glob
 DATA_VERSION = 1
 FILE_PREFIX = "v{}_".format(DATA_VERSION)
 VERSION_FILE_NAME = "VERSION"
@@ -9,19 +9,27 @@ def assert_version(data_directory):
     version_file = os.path.join(data_directory, VERSION_FILE_NAME)
 
     try:
-        assert os.path.exists(version_file), "more"
-        with open(version_file, 'r') as f:
-            try:
-                txt = int(f.read())
-            except FileNotFoundError:
-                raise AssertionError("less")
-            except Exception as e:
-                print('VERSION number not found in data folder')
-                raise e
-            current_version = txt
+        if os.path.exists(version_file):
+            1/0
+            
 
-        assert DATA_VERSION <= txt, "more"
-        assert DATA_VERSION >= txt, "less"
+            assert os.path.exists(version_file), "more"
+            with open(version_file, 'r') as f:
+                try:
+                    txt = int(f.read())
+                except FileNotFoundError:
+                    raise AssertionError("less")
+                except Exception as e:
+                    print('VERSION number not found in data folder')
+                    raise e
+                current_version = txt
+
+            assert DATA_VERSION <= txt, "more"
+            assert DATA_VERSION >= txt, "less"
+        else:
+            fs = (glob.glob(os.path.join(data_directory, '*-*/*')))
+            for f in fs:
+                assert_prefix(f)
     except AssertionError as e:
         _raise_error(e, data_directory)
 

